controls/radgridview/performance/tips-tricks.md

```diff
@@ -31,7 +31,7 @@ __RadGridView__ API allows you to tweak your application for optimal performance
 
 * Avoid using converters. Calling a converter is a time-consuming operation and this will slow down the performance.
 
-* Avoid using __RowStyle__ and/or __AlternateRowStyle__ - use [implicit styles]({%slug styling-apperance-implicit-styles-overview%}) instead. 
+* Avoid applying a __RowStyle__ and/or an __AlternateRowStyle__ that modifies the ControlTemplate that the GridViewRow is created with. This will result in a performance hit.

```

Comment:
The phrase "the ControlTemplate with which the GridViewRow is created" sounds too complicated to me. Can't we just use "the default GridViewRow control template"?